calico/reference/resources/felixconfig.mdx

```diff
@@ -64,6 +64,7 @@ spec:
 | ipv6Support                        | IPv6 support for Felix                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | true, false                                                                                             | boolean                                 | `true`                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| KubeNodePortRanges                 | This parameter holds a list of port ranges used for service node ports. Only used if felix detects kube-proxy running in IPVS mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | string                                                                                                    | Comma-delimited list of port ranges     | `30000:32767`                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| kubeNodePortRanges                 | This parameter holds a list of port ranges used for service node ports. Only used if felix detects kube-proxy running in IPVS mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | string                                                                                                    | Comma-delimited list of port ranges     | `30000:32767`                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Naming convention is different in the felixconfig resource.

Reply:
Oh, it's not even a comma-delimited string there either; it's a list of port ranges.

Reply:
https://github.com/projectcalico/calico/blob/master/api/pkg/apis/projectcalico/v3/felixconfig.go#L296

Reply:
Changed "Comma-delimted list of port ranges" to "List of port ranges"
Changed "KubeNodePortRanges" to "kubeNodePortRanges"

Reply:
Do you want me to make these changes in the older versions as well? 3.23 and 3.24?

Reply:
@fasaxc Are these changes fine?